source/libtomo/accel/gpu/sirt.cu

```diff
@@ -152,7 +153,7 @@ sirt_gpu_compute_projection(data_array_t& gpu_data, int p, int dy, int dt, int d
         CUDA_CHECK_LAST_STREAM_ERROR(stream);
 
         // synchronize the stream (do this frequently to avoid backlog)
-        stream_sync(stream);
+        // stream_sync(stream);
```

Comment:
Also this one?

Reply:
Same response as mlem and the aformentioned `__npp_stream_sync` impl.